app/commands/admin.py

```diff
@@ -0,0 +1,113 @@
+from app.common.database.repositories import users
+from app.common.database.objects import DBUser
+from app.objects import Context
+
+import app
+
+@app.session.commands.register(['restrict', 'unrestrict'], roles=['Admin'])
```

Comment:
I think I would separate the restrict/unrestrict commands. The list should only be for alternative command names, like `ban`, `unban`, etc...